package-lock.json

```diff
@@ -2875,9 +2875,9 @@
       }
     },
     "@bbc/psammead-grid": {
```

Comment:
We should bump the version in the ` package-lock.json` since it is currently `2.0.78`